libraries/lib-import-export/EditRiffTags.cpp

```diff
@@ -3,14 +3,14 @@
 
 #include <array>
 #include <cassert>
+#include <cstdio>
+#include <cstring>
 #include <memory>
 
 namespace LibImportExport
 {
-// Unfortunately, libsndfile has support for neither writing loop info (although
-// it does have support for reading it) nor writing LIST chunks.
-
-void AddMusicInfo(const LibFileFormats::AcidizerTags& acidTags, SNDFILE* file)
+void AddAcidizerTags(
+   const LibFileFormats::AcidizerTags& acidTags, SNDFILE* file)
```

Comment:
I would make `file` a reference.  Otherwise it needs a null check to make me confident.  It is just passed to a sndfile function, which is C style, but I don't know whether the library does null checks or would crash.  Better to use C++ style in this wrapper level.

Reply:
Reworked that utility code, this isn't an issue anymore.